spec/stored_queries/data_sources/data_sources_get_data_sources.yaml

```diff
@@ -0,0 +1,22 @@
+# Get 0 or more data sources matching the given type, optionally
+# filtered by an array of namespace ids
+name: data_sources_get_data_sources
```

Comment:
Can you write a couple of quick tests for these queries?

Nice example tests: https://github.com/kbase/relation_engine/blob/develop/spec/test/stored_queries/test_ontology.py

Reply:
There are integration tests now, which exercises the importer and check the state of the db afterwards with re api and direct Arango calls. The queries are in those code pathways.